src/rc/start-stop-daemon.c

```diff
@@ -712,6 +712,17 @@ start_stop_daemon(int argc, char **argv)
 		if (sscanf(tmp, "%d", &nicelevel) != 1)
 			eerror("%s: invalid nice level `%s' (SSD_NICELEVEL)",
 			    applet, tmp);
+		if ((tmp = getenv("SSD_IONICELEVEL"))) {
```

Comment:
Good point. All the more reason to *never* omit optional curly braces around flow control statements.